src/Controls/src/Core/Label.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.Maui.Controls
 {
 	/// <include file="../../docs/Microsoft.Maui.Controls/Label.xml" path="Type[@FullName='Microsoft.Maui.Controls.Label']/Docs/*" />
 	[ContentProperty(nameof(Text))]
-	public partial class Label : View, IFontElement, ITextElement, ITextAlignmentElement, ILineHeightElement, IElementConfiguration<Label>, IDecorableTextElement, IPaddingElement
+	public partial class Label : View, IFontElement, ITextElement, ITextAlignmentElement, ILineHeightElement, IElementConfiguration<Label>, IDecorableTextElement, IPaddingElement, IEnumerable
```

Comment:
Not sure Label (and similar) are useful? Maybe these "primitive content" ones don't need this? And maybe Span too?

Reply:
this can be useful when you are writing very long text

```cs
new Label
{
	$"Lorem Ipsum is simply dummy text of the printing and typesetting " +
	$"industry. Lorem Ipsum has been the industry's standard dummy text ever " +
	$"since the 1500s, when an unknown printer took a galley of type and scrambled " +
	$"it to make a type specimen book. It has survived not only five centuries, " +
	$"but also the leap into electronic typesetting, remaining essentially " +
	$"unchanged. It was popularised in the 1960s with the release of Letraset " +
	$"sheets containing Lorem Ipsum passages, and more recently with desktop " +
	$"publishing software like Aldus PageMaker including versions of Lorem Ipsum."
},
```